blimflix/client/app/directives/movie-item/movie-item.directive.html

```diff
@@ -0,0 +1,6 @@
+<div class="col-md-2 movie-item-poster">
+  <img src="{{movieItem.poster}}" alt="{{movieItem.title}}" />
+  <div class="details-overlay">
```

Comment:
Mike, when the user clicks in the dark area, the directive should call/display the details directive.
Let's add that behavior, even when the other directive is not ready, let's use a console.log() in the meantime.